CCUF-Chater-and-Policies

```diff
@@ -39,14 +41,14 @@ CCUF Rules
 3.5.	The member organizations that comprise the CCUF approve the Principles and Rules; the CCUF MG approves the Policies and Procedures.
 3.6.	The Management Group will vote on proposed policies and procedures which must pass by simple majority and inform the general CCUF membership.
 3.7.	MG Composition
-3.7.1.	The Management Group (MG) consists of seven elected members from the CCUF membership and one liaison from the CC Development Board (CCDB).  
+3.7.1.	The Management Group (MG) consists of at least three and no more than seven elected members from the CCUF membership and one liaison from the CC Development Board (CCDB).  
 3.7.2.	Because the CCDB is represented by a liaison to the CCUF MG, CCUF members from CCRA member organizations are not eligible to serve on the CCUF MG. [Footnote 1: The list of CCRA Member organizations is at http://www.commoncriteriaportal.org/ccra/members/.]
 3.7.3.	There shall not be more than 2 CCUF MG members from any one organization.
 3.7.4.	Membership of the CCUF MG is on a personal basis rather than an organizational basis.  Thus any CCUF MG member who changes to another organization may remain a CCUF MG member, subject to not breaching the “no more than 2 individuals from one organisation” rule (3.7.3).
 3.7.5.	The MG roles are a Chair, a Vice-Chair, and other roles as decided by the MG. There may also be additional roles appointed by the Management Group for CCUF members.  Appointed roles do not vote on matters before the MG.
 3.8.	MG Elections 
 3.8.1.	There are no requirements to be elected to the CCUF MG other than the willingness to operate in accordance with the CCUF Principles.
-3.8.2.	Members of the Group are elected for a 2 year term of office, with elections held in April of even calendar years for all 7 seats. [Footnote 2: This transition was done to eliminate the disruption of holding an election every year.]
+3.8.2.	Members of the Group are elected for a 2 year term of office, with elections held in April/May of even calendar years for all 7 seats. [Footnote 2: This transition was done to eliminate the disruption of holding an election every year.]
```

Comment:
Maybe this should just be spring instead of April/May? I know that is a larger time block, but maybe that would be a better description?